examples/advanced_circuits_algorithms/Deutsch_Jozsa_Algorithm/Deutsch Jozsa Algorithm.ipynb

```diff
@@ -0,0 +1,1037 @@
+{
```

Comment:
<p>possible combinations, where n is the number of qubits you've initialized.&nbsp;</p><p> </p><p>If n = 3, then we have 8 different possible inputs. We then determine if all of these inputs result in a function that is constant, balanced or neither, so we can compare it with the results we obtain when we run the quantum version of this algorithm. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/177/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>